src/components/AddressInfo/index.tsx

```diff
@@ -39,10 +39,10 @@ const StyledBlock = styled(Block)`
 interface Props {
   safeName?: string
   safeAddress: string
-  ethBalance?: string
+  ethBalance?: number | string
 }
 
-const AddressInfo = ({ ethBalance, safeAddress, safeName }: Props) => {
+const AddressInfo: React.FC<Props> = ({ ethBalance, safeAddress, safeName }: Props) => {
```

Comment:
duplicated Props type definition

also, when the component doesn't accept children as a prop it's better to use JSX.Element, see https://github.com/facebook/create-react-app/pull/8177